### PR TITLE
three fixes

### PR DESCRIPTION
**Pull request by brendankenny**
unrelated fixes (probably easier to look at individual commits, sorry) to things I ran into while integrating lighthouse with an existing CI:
  - ~~switch back to continuous builds now that Linux dev is on m52~~
  - mock out `WebInspector.ConsoleMessage` for a code path hit when resource type and mime type don't match and `WebInspector` wants to log about it
  - get log levels working again. `npmlog.log` is bound to `npmlog`, so can be called independently, but the log `level` can only be set on the parent `npmlog` object, so it has to be exposed.

### CODE REVIEW

File: scripts/download-chrome.sh [hunk 1]
```diff
@@ -12,5 +12,5 @@ if [ -e "$LIGHTHOUSE_CHROMIUM_PATH" ]
 then
   echo "cached chrome found"
 else
-  wget 'https://download-chromium.appspot.com/dl/Linux_x64?type=snapshots' -O chrome.zip && unzip chrome.zip
+  wget 'https://download-chromium.appspot.com/dl/Linux_x64?type=continuous' -O chrome.zip && unzip chrome.zip
```
Comment (paulirish) [inline scripts/download-chrome.sh]: We can't do this yet. LKGR builds have been 3 weeks old at some points. :/

**Comment by paulirish** [comment]
Good catch on log.level failing.  

Revert the continuous change. Otherwise, these are great.

File: scripts/download-chrome.sh (hunk 1 above)
Comment (brendankenny) [inline scripts/download-chrome.sh]: done

**Comment by paulirish** [comment]
Oh classy rebase skills. Well done.

**Pull request merged**